website/docs/features/responsible_ai/Data Balance Analysis.md

```diff
@@ -19,7 +19,7 @@ Fairness-related harms include:
 * **Quality of service harms**: When an AI system does not work as well for one group of people as it does for another.
 * **Stereotyping harms**: When an AI system makes unfair generalizations about groups of people and reinforces negative stereotypes.
 * **Demeaning harms**: When an AI system is actively derogatory or offensive.
-* **Over- and underrepresentation harms**: When an AI system over- or underrepresents some groups of people or may even erase some groups entirely.
+* **Over or under representation harms**: When an AI system over or under represents some groups of people or may even erase some groups entirely.
```

Comment:
underrepresentation is a word, so not sure what acrolinx is complaining about.  perhaps just the dash?  Maybe it should just be "over or underrepresentation". Or maybe "Over/underrepresentation". I don't think it should be 2 words ("under representation").